### DISCUSSION TITLE
why default created_at migration change to current time on update

### DISCUSSION THREAD

**Discussion by tsulatsitamim** [adonisjs/core#4254] [Help]
![Screenshot 2023-09-23 at 22 02 59](https://github.com/adonisjs/core/assets/5946791/46146087-0aae-4bbc-b192-46aa63282967)

every time i save/update my model, the created_at column change to current time. my db is mariadb.
my migration: `table.timestamp('created_at', { useTz: true })`

**Comment by RomainLanz** [comment]
Related issue https://github.com/adonisjs/lucid/issues/924
